versions.json

```diff
@@ -97,7 +97,6 @@
   "v1.40": "v1.40",
   "v1.41": "v1.41",
   "v1.42": "v1.42",
-  "1.43": "1.43",
   "v1.43": "v1.43",
```

Comment:
Is still appears (it was here twice)...
@idanovo is this on purpose?

Reply:
When released version v1.43.0, I accidentally created a tag with the name 1.43.0 instead of v1.43.0.
This is why we have this here.
V1.43.0 should remain, release & tag 1.43.0 are not really exists